<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de tâches</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <property name="leftMargin">
     <number>16</number>
    </property>
    <property name="topMargin">
     <number>16</number>
    </property>
    <property name="rightMargin">
     <number>16</number>
    </property>
    <property name="bottomMargin">
     <number>16</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Gestionnaire de tâches</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnNouveau">
        <property name="text">
         <string>+ Nouvelle tâche</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="filterLayout">
      <item>
       <widget class="QLabel" name="labelFiltre">
        <property name="text">
         <string>Filtrer par état :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboFiltre">
        <item>
         <property name="text">
          <string>Tous</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>À faire</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>En cours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Réalisé</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Abandonné</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>En attente</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelTri">
        <property name="text">
         <string>Trier par :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboTri">
        <item>
         <property name="text">
          <string>Titre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Date début</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Date fin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>État</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listeTaches">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="actionsLayout">
      <item>
       <widget class="QPushButton" name="btnVoir">
        <property name="text">
         <string>Voir détails :D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnModifier">
        <property name="text">
         <string>Modifier ;)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCloturer">
        <property name="text">
         <string>Clôturer :O</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSupprimer">
        <property name="text">
         <string>Supprimer :'(</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
